<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-A2R1E8A" failures="1" tests="1" name="tests.VerifydirectorswhoservedduringtheyearappearingoncompanyinformationpageFA69716_DefaultProfile" time="118.446" errors="0" timestamp="2021-08-11T12:55:53 IST" skipped="0">
  <testcase classname="tests.VerifydirectorswhoservedduringtheyearappearingoncompanyinformationpageFA69716_DefaultProfile" name="GoToUrlAssertUrlPositiveEnterUsernamePositiveEnterPasswordClickLoginButtonPositiveEnterAgentNa" time="118.446">
    <failure message="Unable to locate object: By.xpath: //LI[@id=&#039;ctl00_SideMenu1_liSettings&#039;]/A" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Unable to locate object: By.xpath: //LI[@id='ctl00_SideMenu1_liSettings']/A
at org.testng.Assert.fail(Assert.java:97)
at pages.Verify_directors_who_served_during_the_year_appearing_on_company_information_pageFA69716.Click_Settings2(Verify_directors_who_served_during_the_year_appearing_on_company_information_pageFA69716.java:930)
at tests.VerifydirectorswhoservedduringtheyearappearingoncompanyinformationpageFA69716_DefaultProfile.GoToUrlAssertUrlPositiveEnterUsernamePositiveEnterPasswordClickLoginButtonPositiveEnterAgentNa(VerifydirectorswhoservedduringtheyearappearingoncompanyinformationpageFA69716_DefaultProfile.java:145)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- GoToUrlAssertUrlPositiveEnterUsernamePositiveEnterPasswordClickLoginButtonPositiveEnterAgentNa -->
  <system-out/>
</testsuite> <!-- tests.VerifydirectorswhoservedduringtheyearappearingoncompanyinformationpageFA69716_DefaultProfile -->
